lib/msal-browser-poc/package.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "msal-browser",
```

Comment:
We need to decide if we want this or `@azure/msal-browser` (which is probably better for branding/consistency)

Reply:
We should decide on this in PR #1108.